javaparser-core/src/main/java/com/github/javaparser/printer/concretesyntaxmodel/CsmConditional.java

```diff
@@ -93,6 +94,17 @@ boolean evaluate(Node node, ObservableProperty property) {
                 return property.getValueAsBooleanAttribute(node);
             }
         }
+        , IS_STMT_GROUP_ENTRY {
+            @Override
```

Comment:
Can you explain the purpose of this change in more detail?

Reply:
Please see above.